### DISCUSSION TITLE
HOW DUMB I AM

### DISCUSSION THREAD

**Discussion by NEW-BOOTY** [skills/.github#542] [Introduction to GitHub]
IVE TIRED TO CREATE A REPOSITORIES IN THE PASS NEVER KNEW IT WAS KIND OF THAT EASY. DUMB DUMB DUMB DUUUMB.

**Comment by DresIsNotReal** [comment]
If ür sö lääääzy, why did ü type ääääl this text?

**Reply by NEW-BOOTY** [reply]
don't remember doing it.
